README.md

```diff
@@ -645,6 +645,17 @@ You can view an example config file with all available settings [here](config.ex
 
 By default, without specifying `--config` the agent will try to read `~/.holmes/config.yaml`. When settings are present in both config file and cli, the cli option takes precedence.
 
+
+When running **HolmesGPT in a Kubernetes cluster**, you can securely pass configuration settings using a **Kubernetes Secret**. The secret will be mounted into the pod at `~/.holmes/config.yaml`:
```

Comment:
I thought we wanted the whole config to be the secret, 
doesnt make sense to only do the opensearch, since we need this for other parts of the config that cant be loaded via env var like grafana/loki , thats why we did this instead